src/main/scala/com/metl/model/Reputation.scala

```diff
@@ -25,9 +22,9 @@ object Reputation extends Logger{
   def addStanding(who:String,what:GainAction.Value):Unit = Stopwatch.time("Reputation:addStanding(%s,%s)".format(who,what),{
     val score = Informal.value(what)
     if (score > 0){
-      val currentStanding = standingMap(who)
+      val currentStanding = standingMap.getOrElseUpdate(who, Informal.standing(who))
```

Comment:
uh, what's the benefit of getOrElseUpdate here, given that you're about to change it?  The update is about to be thrown away anyway, isn't it, in favour of the new one?